contracts/TreasuryExtender.sol

```diff
@@ -0,0 +1,282 @@
+pragma solidity ^0.8.10;
+
+// interfaces
+import "./interfaces/IERC20.sol";
+import "./interfaces/ITreasury.sol";
+import "./interfaces/IAllocator.sol";
+import "./interfaces/ITreasuryExtender.sol";
+
+// types
+import "./types/OlympusAccessControlledImproved.sol";
+
+// libraries
+import "./libraries/SafeERC20.sol";
+
+contract TreasuryExtender is OlympusAccessControlledImproved, ITreasuryExtender {
```

Comment:
What was the reason for changing from `TreasuryAdapter` to `TreasuryExtender`? Seems adapter is more appropriate, from a SE perspective.